### PR TITLE
[macsecorch]: Change SCI in appdb for better readability

### PR DESCRIPTION
**Pull request by Pterosaur**
Signed-off-by: Ze Gan <ganze718@gmail.com>

Waiting for PR: https://github.com/Azure/sonic-wpa-supplicant/pull/48

**What I did**
To change SCI in appdb for better readability.

**Why I did it**
The original SCI was like `Ethernet0:72068799825728594`, The new SCI is like `Ethernet0:5254000D310A0001`

**How I verified it**
Check Azp.

**Details if related**

### CODE REVIEW

File: orchagent/macsecorch.cpp [hunk 1]
```diff
@@ -218,6 +220,68 @@ static void lexical_convert(const std::string &buffer, MACsecAuthKey &auth_key)
...
+        SWSS_LOG_ENTER();
+
+        return m_sci;
+    }
+
+    std::string str() const
+    {
+        SWSS_LOG_ENTER();
+
+        return boost::algorithm::to_lower_copy(swss::binary_to_hex(&m_sci, sizeof(m_sci)));
```
Comment (jimmyzhai) [inline orchagent/macsecorch.cpp:237]: Is using `boost::algorithm::to_lower` not ok?

Comment (Pterosaur) [reply orchagent/macsecorch.cpp:237]: to_lower is in-place conversion.

**Review by jimmyzhai — APPROVED** [review]

File: orchagent/macsecorch.cpp [hunk 2]
```diff
@@ -218,6 +220,68 @@ static void lexical_convert(const std::string &buffer, MACsecAuthKey &auth_key)
...
+    }
+
+private:
+    sai_uint64_t m_sci;
+};
+
+namespace swss {
+
+template<>
+inline void lexical_convert(const std::string &buffer, MACsecSCI &sci)
```
[the remark this answers is not in the document]
Comment (Pterosaur) [reply orchagent/macsecorch.cpp:268]: Offline discussed

**Review by jimmyzhai — APPROVED** [review]

**Comment by prsunny** [comment]
@Pterosaur , could you please mention the reason for PR merge bypassing coverage. We would want to track the coverage issues and work on the fix if it is an infra issue.

**Comment by Pterosaur** [comment]
> @Pterosaur , could you please mention the reason for PR merge bypassing coverage. We would want to track the coverage issues and work on the fix if it is an infra issue.

Mark the reason for bypassing coverage:
The new codes should be covered by test: https://github.com/Azure/sonic-swss/blob/master/tests/test_macsec.py
Meanwhile, according to the hint of coverage, it's impossible that the line 244 was covered but line 242 was ignored.
MicrosoftTeams-image
So, I bypass the coverage checker.

**Pull request merged**